packages/apps-sdk/src/sdk.ts

```diff
@@ -31,9 +34,21 @@ export class SDK {
    */
   private configureApp(): void {
     // First we obtaing the id of this app from the url
-    const parsed = queryString.parse(window.location.search);
+    const parsed = qs.parse(window.location.search.substring(1), { decoder });
     const parsedAppName = Array.isArray(parsed.appName) ? parsed.appName[0] : parsed.appName;
+    const parsedQueryParams = Array.isArray(parsed.queryParams) ? parsed.queryParams[0] : parsed.queryParams;
+    const parsedInitialData = Array.isArray(parsed.initialData) ? parsed.initialData[0] : parsed.initialData;
+
     this.appName = parsedAppName ? parsedAppName : 'messenger-iframe';
+    this.queryParams = parsedQueryParams;
```

Comment:
Why not to assign this value and the next one at once and not to use this duplication?